docs_app/content/guide/operators.md

```diff
@@ -0,0 +1,219 @@
+
+```js
+const multiplyByTen = () => (source) => {
+  return new Observable((subscriber) => {
+    source.subscribe({
+      next(value) { subscriber.next(10 * value); },
+      error(err) { subscriber.error(err); },
+      complete() { subscriber.complete() },
+    });
+  });
+};
+
+const input = from([1, 2, 3, 4]);
+const output = input.pipe(multiplyByTen());
+output.subscribe(x => console.log(x));
+```
+
+Which outputs:
+
+```none
+10
+20
+30
+40
+```
+
+Notice that a subscribe to `output` will cause the `input` Observable to be subscribed. We call this an "operator subscription chain".
+
+You can also create custom operators by wrapping calls to existing operators. We can reimplement our `multiplyByTen` operator by wrapping the `map` operators which will project our multiplicatoin function.
+
+```js
+import { map } from 'rxjs/operators';
+
+const multiplyByTen = () => map((value) => 10 * value);
+```
+
+## Creation Operators
+
+**What are creation operators?** Besides pipeable operators, creation operators are functions that can be used to create an Observable with some common predefined behavior or by combining other Observables.
+
+A typical example of a creation operator would be the `interval` function. It takes a number (not an Observable) as input argument, and produces an Observable as output:
+
+```js
+const observable = interval(1000 /* number of milliseconds */);
+```
+
+See the list of [all static creation operators here](#creation-operators).
+
+Some *Combination Operators* such as `merge`, `combineLatest`, `concat`, etc. take *multiple* Observables as input, not just one, for instance:
+
+```js
+const observable1 = interval(1000);
+const observable2 = interval(400);
+
+const merged = merge(observable1, observable2);
+```
+
+## Marble diagrams
+
+To explain how operators work, textual descriptions are often not enough. Many operators are related to time, they may for instance delay, sample, throttle, or debounce value emissions in different ways. Diagrams are often a better tool for that. *Marble Diagrams* are visual representations of how operators work, and include the input Observable(s), the operator and its parameters, and the output Observable.
+
+<span class="informal">In a marble diagram, time flows to the right, and the diagram describes how values ("marbles") are emitted on the Observable execution.</span>
+
+Below you can see the anatomy of a marble diagram.
+
+<img src="assets/images/examples/marble-diagram-anatomy.svg">
+
+Throughout this documentation site, we extensively use marble diagrams to explain how operators work. They may be really useful in other contexts too, like on a whiteboard or even in our unit tests (as ASCII diagrams).
+
+## Categories of operators
+
+There are operators for different purposes, and they may be categorized as: creation, transformation, filtering, combination, multicasting, error handling, utility, etc. In the following list you will find all the operators organized in categories.
```

Comment:
I really like this overview, but I fear that we could forget updating this. Therefore I would recommend adding a sentence like 'For a complete api overview, see references page' something like that